src/theme/templates/home-layout.hbs

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta {{that}}>
+  <title>{{title}}</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="author" content="Troy Forster" />
```

Comment:
The trailing space and forward slash is not necessary in HTML5 and can be simplified slightly like the above.  I also see them on the stylesheet links.  Doesn't break or change anything, but more for consistency.  I can make note of it in our style guides as well.  Hopefully we can have these automatically addressed when we create the linting/formatting rules.